frontend/src/containers/Board.js

```diff
@@ -122,7 +122,7 @@ class Board extends Component {
       addCard(board.boardId, targetLaneId, card.id, card.index);
 
       if (this.sideNav) {
-        this.sideNav.removeCard(card.id);
+        this.sideNav.instanceRef.refreshView(board.boardId);
```

Comment:
IMHO you shall wait for addCard operation to finish to increase the chance of refreshing AFTER it was changed in backend.

But now that I am looking at the code, I believe the quickest workaround is to make the removeCard work, i.e.
```javascript
this.sideNav.removeCard(card.id);
```

Reply:
Tried that and it does not work as it should. More debugging and more time needed to make it work with removeCard.

Reply:
moved this to be triggered with the ws event